src/transformState.js

```diff
@@ -63,6 +63,23 @@
  */
 function transformState(state, transforms) {
   // write code here
+  for (const i of transforms) {
```

Comment:
transforns - plural
transform - singular
```suggestion
  for (const transform of transforms) {
```